geode-core/src/distributedTest/java/org/apache/geode/internal/cache/ClearTXLockingDUnitTest.java

```diff
@@ -330,46 +339,46 @@ private void checkForConsistencyErrors(String regionName) {
     for (int i = 0; i < NUMBER_OF_PUTS; i++) {
       String theKey = regionName + THE_KEY + i;
       if (r0Contents.containsKey(theKey)) {
-        softly.assertThat(r1Contents.get(theKey))
+        assertThat(r1Contents.get(theKey))
```

Comment:
After looking through the test code, I think the race is possibly caused by the region.clear is distributed. clear occurred on one node may not occur on the other node.

I think you can add a wait for some time to see the clear is distributed to vm0 for region2. Do you want to add it and see the stress test can pass?